### ISSUE TITLE
Cannot startx with kernel 4.2.0-30 

### ISSUE THREAD

**Issue by myitcv** [rasa/vmware-tools-patches#94]
* Kernel `4.2.0-30`
* Ubuntu 15.10
* Latest patches

After upgrading from `4.2.0-27` to `4.2.0-30` I am unable to `startx`. Everything freezes - although telling the now 'frozen' VM to 'shutdown' from the menu works.

Anyone else seeing this?

**Comment by philthompson10** [comment]
Yes. Creating a new VM, upgrading to that kernel and then installing open-vm-tools seems to be Ok.

Any ideas on how to rescue the 'frozen' VM?
